.gitignore

```
@@ -55,3 +55,6 @@ built_pkgs
 __pycache__
 .circleci/config.yml-local
 .env
+compile_commands.json
```

```suggestion
compile_commands.json
compile_commands.sqlite3
```

or maybe `compile_commands.*`? (`bear` creates the sqlite database during builds, but apparently cleans it up afterwards)

---

Done. Replaced it with `compile_commands.*`.